mp/src/game/shared/momentum/mom_gamemovement.cpp

```diff
@@ -1073,11 +1073,11 @@ bool CMomentumGameMovement::CheckJumpButton()
     {
         if (player->m_Local.m_bDucking)
         {
-            mv->m_vecVelocity[2] = flGroundFactor * sqrt(2.f * GetCurrentGravity() * 45.0f); // 2 * gravity * height
+            mv->m_vecVelocity[2] = flGroundFactor * 289.0f;
         }
         else
         {
-            mv->m_vecVelocity[2] += flGroundFactor * sqrt(2.f * GetCurrentGravity() * 45.0f); // 2 * gravity * height
+            mv->m_vecVelocity[2] += flGroundFactor * 289.0f;
```

Comment:
This change may be problematic in other gravity situations than 800. Any reason for this optimization? I understand sqrt is not something you want to run too often but given gravity could change in the future, we may want to keep it.

Reply:
TF2 uses 289 as a hardcoded jump factor that doesn't depend on the gravity. It is possible to add a dependence on gravity like in the other gamemodes by using 289^2/(2*800) instead of 45 to still get the desired factor when the gravity is 800.

Reply:
Ah okay. Forgot to expand the diff to see if this was under a gamemode check already and it was, it's good then, just the tab -> space thing and we can merge it.

Reply:
Already did the 289^2/(2*800) thing, since it still gives exactly 289 as needed with 800 gravity using it.